UI/window-basic-auto-config.cpp

```diff
@@ -566,7 +566,8 @@ void AutoConfigStreamPage::UpdateKeyLink()
 	if (serviceName == "Twitch") {
 		streamKeyLink =
 			"https://www.twitch.tv/broadcast/dashboard/streamkey";
-	} else if (serviceName == "YouTube / YouTube Gaming") {
+	} else if ((serviceName == "YouTube-RTMP") or
```

Comment:
Please add spaces before and after "-", i.e. YouTube - RTMP, to be consistent with the names for other services. Same for HLS, and also need to change in services.json.